### PR TITLE
Fix image paths

### PR DESCRIPTION
**Pull request by karlhorky**

### CODE REVIEW

**Comment by kingthorin** [comment]
The paths need to be relative to the permalink in the front matter of the pages using them.

Are these images not displaying for you when you browse the pages?

File: pages/attacks/Content_Spoofing.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ is properly done by the victim business. For public traded companies,
 its shares will be falling down, leading to uncontrolled loss of
 millions.
 
-![](../assets/images/Fake-text.png "Fake-text.png")
+![](assets/images/Fake-text.png "Fake-text.png")
```
Comment (kingthorin) [inline pages/attacks/Content_Spoofing.md]: This one seems fine:
image

**Comment by karlhorky** [comment]
They are not being displayed, correct.

I did this because [this page](https://owasp.org/www-community/Types_of_Cross-Site_Scripting) has images working with the relative paths:

https://github.com/OWASP/www-community/blame/master/pages/Types_of_Cross-Site_Scripting.md#L104

File: pages/attacks/Reverse_Tabnabbing.md [hunk 2]
```diff
@@ -45,13 +45,13 @@ javascript function.
 Link between parent and child pages when prevention attribute is not
 used:
 
-![](../../assets/images/TABNABBING_OVERVIEW_WITH_LINK.png "File:TABNABBING_OVERVIEW_WITH_LINK.png")
+![](assets/images/TABNABBING_OVERVIEW_WITH_LINK.png "File:TABNABBING_OVERVIEW_WITH_LINK.png")
```
Comment (kingthorin) [inline pages/attacks/Reverse_Tabnabbing.md]: This one isn't working it should only be a single traversal.
```suggestion
![](../assets/images/TABNABBING_OVERVIEW_WITH_LINK.png "File:TABNABBING_OVERVIEW_WITH_LINK.png")
```

**Comment by karlhorky** [comment]
But! I guess that's because this page is in the root of pages. Fixing now...

**Comment by karlhorky** [comment]
Fixed!

File: pages/attacks/Reverse_Tabnabbing.md [hunk 3]
```diff
@@ -45,13 +45,13 @@ javascript function.
...
 
 ### Without back link
 
 Link between parent and child pages when prevention attribute is used:
 
-![](../../assets/images/TABNABBING_OVERVIEW_WITHOUT_LINK.png "File:TABNABBING_OVERVIEW_WITHOUT_LINK.png")
+![](assets/images/TABNABBING_OVERVIEW_WITHOUT_LINK.png "File:TABNABBING_OVERVIEW_WITHOUT_LINK.png")
```
Comment (kingthorin) [inline pages/attacks/Reverse_Tabnabbing.md]: Same here
```suggestion
![](../assets/images/TABNABBING_OVERVIEW_WITHOUT_LINK.png "File:TABNABBING_OVERVIEW_WITHOUT_LINK.png")
```

File: pages/vulnerabilities/Insufficient_Session-ID_Length.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ The WebLogic deployment descriptor should specify a session identifier length of
 
 The expected number of seconds required to guess a valid session identifier is given by the equation:
 
-![](../../assets/images/session_id_guessing.gif)
+![](assets/images/session_id_guessing.gif)
```
Comment (kingthorin) [inline pages/vulnerabilities/Insufficient_Session-ID_Length.md]:
```suggestion
![](../assets/images/Session_id_guessing.gif)
```

File: pages/vulnerabilities/Insufficient_Session-ID_Length.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ The WebLogic deployment descriptor should specify a session identifier length of
 
 The expected number of seconds required to guess a valid session identifier is given by the equation:
 
-![](../../assets/images/session_id_guessing.gif)
+![](../assets/images/session_id_guessing.gif)
```
Comment (kingthorin) [inline pages/vulnerabilities/Insufficient_Session-ID_Length.md:22]:
```suggestion
![](../assets/images/Session_id_guessing.gif)
```

**Comment by karlhorky** [comment]
Ah, sorry about the error in the image name, thanks for fixing.

Really cool resources that are on this website. I'll probably find myself contributing here more often.

**Comment by kingthorin** [comment]
No problem at all, thanks for your help.

There's always new stuff to add or things to touch-up following the migration. We can use any help you feel like providing.

**Pull request merged**